src/server/routes/flight.js

```diff
@@ -25,4 +26,8 @@ router.route('/:flightname/telemetry')
 router.route('/:flightname/telemetry')
   .post(validate(paramValidation.telemetry), telemetryCtrl.postTelemetry);
 
+/** GET flight telemetry */
+router.route('/:flightname/history')
```

Comment:
Do we want jwtToken for this endpoint.
If yes ,  we just need to add the authentication param like below

/** GET flight */
router.route('/:flightname')
  .get(validate(paramValidation.flightInfo), authentication, flightInfoCtrl.getFlightInfo);